code/datums/diseases/advance/advance.dm

```diff
@@ -194,6 +194,35 @@
 	for(var/i = 1; number_of >= i && possible_symptoms.len; i++)
 		. += pick_n_take(possible_symptoms)
 
+/// Returns the currently focused symptoms, if there are any. Determined by the reagents in the container. Maximum returned symptom amount is dependent on the symptom limit.
+/datum/disease/advance/proc/GetFocusedSymptoms(datum/reagents/container, level_min, level_max)
+	. = list() // The currently focused symptoms, if there are any.
+	if (symptoms.len >= VIRUS_SYMPTOM_LIMIT) // If the symptom cap is reached, we simply return an empty list.
+		return
+	var/list/possible_focuses = list()
+	var/list/reagents_to_consume = list()
+	for(var/symp in SSdisease.list_symptoms)
+		var/datum/symptom/S = new symp
```

Comment:
Do not shorten variable names, rename this to `symptom`.

Reply:
Carried over from the funny generation code.